appIndicator.js

```diff
@@ -303,15 +297,23 @@ class AppIndicators_IconActor extends St.Icon {
 
         this._indicator     = indicator
         this._iconSize      = icon_size
+        this._iconSize0     = icon_size
```

Comment:
Does this make sense?

Reply:
Yes, I suppose you mean:

```diff
--- a/appIndicator.js
+++ b/appIndicator.js
@@ -21,6 +21,7 @@ const Gio = imports.gi.Gio
 const GLib = imports.gi.GLib
 const GObject = imports.gi.GObject
 const Gtk = imports.gi.Gtk
+const Panel = imports.ui.panel;
 const St = imports.gi.St
 const Shell = imports.gi.Shell
 
@@ -347,7 +348,6 @@ class AppIndicators_IconActor extends St.Icon {
 
         this._indicator     = indicator
         this._iconSize      = icon_size
-        this._iconSize0     = icon_size;
         this._iconCache     = new IconCache.IconCache()
         this._cancellable   = new Gio.Cancellable();
         this._loadingIcons  = new Set();
@@ -772,6 +772,6 @@ class AppIndicators_IconActor extends St.Icon {
         if (sizeValue > 0)
             this._iconSize = sizeValue;
         else
-            this._iconSize = this._iconSize0;
+            this._iconSize = Panel.PANEL_ICON_SIZE;
     }
 });
```

This is of course possible, but then if for some unlikely reason
```
this._iconBox = new AppIndicator.IconActor(indicator, Panel.PANEL_ICON_SIZE);
```
is changed it will fail in strange ways. We can of course change that, but maybe the current solution is a bit more robust at some memory cost.